src/app/year-calendar/year-calendar-config.ts

```diff
@@ -6,7 +6,6 @@ export interface IConfig {
   isYearDisabledCallback?: (date: Moment) => boolean;
   allowMultiSelect?: boolean;
   yearFormat?: string;
-  yearFormatter?: (month: Moment) => string;
```

Comment:
why did you remove that?

Reply:
Ah, I think I need more specific instructions ;-) Hope you can point me in the right direction, especially wrt hooking the year calendar up to the month and date calendars. If the other things you point out are minor things, please help me get started :-)